vms/shift/utils.py

```diff
@@ -36,6 +36,9 @@ def create_event_with_details(event):
     Creates and returns event with passed name and dates
     """
     e1 = Event(name=event[0], start_date=event[1], end_date=event[2])
```

Comment:
Instead of modifying attributes afterwards won't it be a better practice to pass it during event init itself.

Reply:
This is the hacky way you suggested in meeting so that only 1 test is changed otherwise we'll have to modify like every test done till now. Also this will anyhow get changed when we implement dictionaries.

Reply:
Why are we not moving to dicts rightaway?

Reply:
Because moving to dictionaries has a lot of dependency and will need existing tests to be completed first so we can tackle all problem at once.

Reply:
You can have a check of len(event) by if else inside which you call the Event constructor.

Reply:
Okay :+1: